File: trunk/download/stream/src/test/java/org/simpleframework/xml/transform/util/CurrencyTransformTest.java
package org.simpleframework.xml.transform.util;

import java.util.Currency;
import java.util.Locale;

import junit.framework.TestCase;

public class CurrencyTransformTest extends TestCase {
   
   public void testCurrency() throws Exception {
      Currency currency = Currency.getInstance(Locale.UK);
      CurrencyTransform format = new CurrencyTransform();
      String value = format.write(currency);
      Currency copy = format.read(value);
      
      assertEquals(currency, copy);
   }
}

File: port/src/main/Xml/Transform/CurrencyTransformTest.cs
#region Using directives
using SimpleFramework.Xml.Transform;
using SimpleFramework.Xml;
using System.Collections.Generic;
using System;
#endregion
package SimpleFramework.Xml.transform;
public class CurrencyTransformTest : TestCase {
   public void TestCurrency() {
      Currency currency = Currency.getInstance(Locale.UK);
      CurrencyTransform format = new CurrencyTransform();
      String value = format.write(currency);
      Currency copy = format.read(value);
      AssertEquals(currency, copy);
   }
}
}
